package.json

```diff
@@ -45,13 +45,33 @@
     "babel-eslint": "^10.1.0",
     "babel-plugin-inline-import": "^3.0.0",
     "eslint": "^6.8.0",
+    "lint-staged": "^10.0.8",
     "gh-pages": "^3.1.0",
     "prettier": "^2.0.5",
     "react": "^16.13.1",
     "react-dom": "^16.2.0",
     "stylelint": "^13.3.2",
     "url-parse": "^1.2.0",
-    "version-bump-prompt": "^6.1.0"
+    "version-bump-prompt": "^6.1.0",
+    "husky": "^7.0.0"
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    "*.{md,html}": [
+      "prettier --write"
+    ],
+    "*.scss": [
+      "prettier --write",
+      "stylelint --fix"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
See my comment above on the `pre-commit` file. I don't know if you need both the `npx lint-staged` command in that file *and* this code block on `package.json`. I could be wrong though!

Reply:
We can remove this block